arxiv/release/tag_check.py

```diff
@@ -0,0 +1,140 @@
+"""Functions to ensure version of git tag is correct for travis and to get that version.
+
+This is intended as a check to use in travis ci.
+
+It will:
+1. Check if there is a git tag on the commit.
+2. Check that the tag is a semver
```

Comment:
I think this answers my previous question on "bad".

Reply:
:+1: